bin/blockfrostUtils.js

```diff
@@ -0,0 +1,74 @@
+
+import { readFile } from 'fs/promises';
+import { BlockFrostAPI } from "@blockfrost/blockfrost-js";
+
+/**
+ * 
+ * @param {*} blockfrostKey 
+ * @param {*} walletAddress 
+ * @returns 
+ */
+const queryUTxOviaBlockfrost = async (blockfrostKey, walletAddress) => {
+  const blockfrost = new BlockFrostAPI({
+    projectId: blockfrostKey,
+  });
+  const utxos = await blockfrost.addressesUtxosAll(walletAddress);
+  let TxHash = "";
+  let TxIx = 0;
+  let Amount = 0;
+  for (const utxo of utxos) {
+    if (utxo.amount.reduce((sum, item) => sum + parseInt(item.quantity, 10), 0) > 1000000) {
+      TxHash = utxo.tx_hash; 
+      TxIx = utxo.output_index;
+      Amount = utxo.amount.reduce((sum, item) => sum + parseInt(item.quantity, 10), 0) 
+    }
+  }
+  return { TxHash, TxIx, Amount };
+}
+
+/**
+ * 
+ * @param {blockfrostKey for the transaction. Beware that blockfrost key is different for different testnets and mainnet} blockfrostKey 
+ * @returns transaction id with which transaction can be found on any cardano explorer.
+ */
+const submitTransactionViaBlockfrost = async (blockfrostKey) => {
+  const blockfrost = new BlockFrostAPI({
+    projectId: blockfrostKey,
+  });
+  const tx = await readFile('./tx.signed', { encoding: 'binary' });
+  const cbor_hex = JSON.parse(tx).cborHex;
+  const transaction = await blockfrost.txSubmit(cbor_hex)
```

Comment:
local variable `transaction` is reduntant
you can simply `return await blockfrost?.txSubmit(cbor_hex);` (added a chain operator in case blockfrost is not what you expect)